pulp_python/app/tasks/sync.py

```diff
@@ -117,30 +121,46 @@ def _fetch_specified_metadata(remote, project_specifiers):
         metadata = json.load(open(downloader.path))
         for version, packages in metadata['releases'].items():
             for package in packages:
-                # If neither specifiers nor digests have been set, then we should add the unit
-                if not project.version_specifier and not digests.exists():
+                if project.name in excludes:
+                    whatnottosync=excludes[project.name]
```

Comment:
```whatnottosync``` is a little unweildy, maybe call it ```do_not_sync```?

Equals sign should have spaces on either side (PEP8).